frontend/src/e2e-test/specs/7_messaging/messaging.spec.ts

```diff
@@ -167,6 +191,36 @@ describe('Sending and receiving messages', () => {
         await messagesPage.assertMessageContent(1, defaultReply)
       })
 
+      test('Citizen sends a message to backup care child', async () => {
+        await insertBackupCareFixtures([
+          {
+            id: uuidv4(),
+            childId,
+            unitId: backupDaycareId,
+            groupId: backupGroupFixtureId,
+            period: {
+              start: LocalDate.todayInSystemTz().formatIso(),
```

Comment:
No mocked time = :cry:

Reply:
The problem lies deeper since `LocalDate.todayInSystemTz()` is used in so many places in our fixtures & tests 😢